build.gradle

```diff
@@ -6,6 +6,9 @@ buildscript {
     maven {
       url 'https://plugins.gradle.org/m2/'
     }
+    maven {
+      url 'https://oss.sonatype.org/content/repositories/snapshots/'
```

Comment:
OK for development, can't commit until this is removed.

Reply:
yep, I added `do not merge` label to the PR